appveyor.yml

```diff
@@ -1,64 +0,0 @@
-
```

Comment:
In any event, I have been pushing for this to get fixed. So am hopeful that we won't have to go through this in the future.